app/upgrades/v22/upgrades.go

```diff
@@ -23,6 +25,13 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// Properly register consensus params. In the process, change params as per:
+		// https://forum.osmosis.zone/t/raise-maximum-gas-to-300m-and-lower-max-bytes-to-5mb/1116
+		defaultConsensusParams := tmtypes.DefaultConsensusParams().ToProto()
+		defaultConsensusParams.Block.MaxBytes = 5000000
+		defaultConsensusParams.Block.MaxGas = 300000000
+		keepers.ConsensusParamsKeeper.Set(ctx, &defaultConsensusParams)
```

Comment:
Need an ack that we are good with all defaults except for the MaxBytes and MaxGas here